packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.html

```diff
@@ -8,14 +8,61 @@
             <span data-translate="vrack__assigned-ipv6_add_subnet"></span>
         </button>
     </div>
+    <!-- subridg -->
+    <li
+        class="vrack__assigned-ipv6__subrang"
+        data-ng-repeat="bridged in $ctrl.bridgedSubranges track by $index"
+    >
+        <span
+            class="vrack__assigned-ipv6__subrang__title"
+            data-ng-bind="::bridged.bridgedSubrange"
+        ></span>
+        <span
+            >SLAAC
```

Comment:
```suggestion
            >$ctrl.label.SLAAC
```
Put this in constant file